contrib/exporters/README.md

```diff
@@ -0,0 +1,190 @@
+```
+FROM ubuntu:21.10
+
+RUN apt-get update \
+  && apt-get install -y gnupg2 apt-transport-https curl
+
+ARG ZCASHD_USER=zcashd
+ARG ZCASHD_UID=2001
+
+########
+# APT
+########
+#ARG ZCASH_SIGNING_KEY_ID=6DEF3BAF272766C0
+#ARG ZCASH_VERSION=
+# The empty string for ZCASH_VERSION will install the apt default version,
+# which should be the latest stable release. To install a specific
+# version, pass `--build-arg 'ZCASH_VERSION=<version>'` to `docker build`.
+
+########
+# APT
+########
+# RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys $ZCASH_SIGNING_KEY_ID \
+#   && echo "deb [arch=amd64] https://apt.z.cash/ stretch main" > /etc/apt/sources.list.d/zcash.list \
+#   && apt-get update
+
+# RUN if [ -z "$ZCASH_VERSION" ]; \
+#     then apt-get install -y zcash; \
+#     else apt-get install -y zcash=$ZCASH_VERSION; \
+#     fi; \
+#     zcashd --version
+
+RUN useradd --home-dir /srv/$ZCASHD_USER \
+            --shell /bin/bash \
+            --create-home \
+            --uid $ZCASHD_UID\
+            $ZCASHD_USER
+
+#######
+# ADD from local resources(e.g. binaries, src, params, etc.)
+#
+# Note: if you use binaries ensure they are compatible with your image
+# FROM ubuntu:21.10
+#######
+ADD zcashd /usr/local/bin
+ADD zcash-cli /usr/local/bin
+ADD zcash-fetch-params /usr/local/bin
+
+USER $ZCASHD_USER
+WORKDIR /srv/$ZCASHD_USER
+RUN mkdir -p /srv/$ZCASHD_USER/.zcash/ \
+    && touch /srv/$ZCASHD_USER/.zcash/zcash.conf
+
+ADD entrypoint.sh /entrypoint.sh
+ENTRYPOINT ["/entrypoint.sh"]
+```
+IMPORTANT NOTE: Ensure whatever you ADD into /usr/local/bin above, is in the same
+path as this Dockerfile:
+
+```
+zcash/contrib/docker/Dockerfile
+zcash/contrib/docker/zcashd
+zcash/contrib/docker/zcash-cli
+zcash/contrib/docker/zcash-fetch-params
+```
+
+docker build . -t localhost:5000/electriccoinco/zcashd:v4.7.0-rc1.1 --no-cache
+
+docker tag electriccoinco/zcashd:v4.7.0-rc1.1 localhost:5000/electriccoinco/zcashd:v4.7.0-rc1.1
+
+docker push localhost:5000/electriccoinco/zcashd:v4.7.0-rc1.1
+
+If you get an error on the push command, remove the image hash for the registry and restart it:
+```
+mdr0id@pop-os:~/Code/nu5_zcash/zcash/contrib/docker$ docker run -d -p 5000:5000 --name registry registry:2
+docker: Error response from daemon: Conflict. The container name "/registry" is already in use by container "76ea0528921aabe83534c0502bb943c75f99ba57aacb58120aba90c74675cf05". You have to remove (or rename) that container to be able to reuse that name.
+See 'docker run --help'.
+
+mdr0id@pop-os:~/Code/nu5_zcash/zcash/contrib/docker$ docker rm 76ea0528921aabe83534c0502bb943c75f99ba57aacb58120aba90c74675cf05
+
+mdr0id@pop-os:~/Code/nu5_zcash/zcash/contrib/docker$ docker run -d -p 5000:5000 --name registry registry:2
+
+```
+
+4. Update your docker-compose.yml to contain your local image name you just pushed:
+
+```
+---
+version: '2'
+
+services:
+  zcashd:
+    image: localhost:5000/electriccoinco/zcashd:v4.7.0-rc1.1
```

Comment:
This should be updated to v5.2.0

Reply:
This is used when you want to profile a locally built hash and not a formal docker image from a registry